protocol/0074-BTCH-batch-market-instructions.md

```diff
@@ -25,7 +25,7 @@ Overall, building the ability to handle batches of market instructions in a sing
   - **Cancellations**: this is a list (repeated field) of Cancel Order instructions
   - **Amendments**: this is a list (repeated field) of Amend Order instructions
   - **Submissions**: this is a list (repeated field) of Submit Order instructions
-- Additionally the batch may contain a single transaction to change the current margin mode.
+- Additionally the batch may contain a single transaction to change the current margin mode. Unlike cancellations/amendments/submissions, if this fails the rest of the batch will be cancelled.
```

Comment:
Should this not only apply to other [submissions|all market instructions] for the same market?